components/ListItem.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import { View, Text, Image, TouchableHighlight } from 'react-native';
+import { StyleSheet } from 'react-native';
+
+const ListItem = ({ imageSource, text, onPress }) => {
+	return (
+		<TouchableHighlight onPress={onPress}>
+			<View style={styles.container}>
+				<Image
+					style={styles.image}
+					source={require('../assets/plus.png')}
```

Comment:
I would move assets like images, colors, fonts, strings, ... etc to a config folder. Inside the folder, you can create an index.js file and export objects that contain all that info to the rest of the app, that way when you want to use/find an image (and you have 200 of them) you can just look it up in this folder. This works well especially if you use the same image more than once. Check any of our react, react-native projects to see more 😛